File: Lecture05/ListViewAdapters/app/src/main/java/com/codingblocks/listviewadapters/MainActivity.java
package com.codingblocks.listviewadapters;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.ListView;

import com.codingblocks.listviewadapters.adapters.CourseListAdapter;
import com.codingblocks.listviewadapters.models.Course;

import java.util.ArrayList;

public class MainActivity extends AppCompatActivity {

    ArrayList<Course> courses = Course.genRandomCourses(100);
    ListView lvCourses;
    CourseListAdapter courseListAdapter;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        lvCourses = findViewById(R.id.lvCourses);
        courseListAdapter = new CourseListAdapter(this, courses);
        lvCourses.setAdapter(courseListAdapter);

    }
}

File: Lecture16/HardwareSensors/app/src/main/java/com/codingblocks/hardwaresensors/MainActivity.kt
package com.codingblocks.hardwaresensors

import android.content.Context
import android.hardware.Sensor
import android.hardware.SensorManager
import android.support.v7.app.AppCompatActivity
import android.os.Bundle

class MainActivity : AppCompatActivity() {
    val TAG = "SENS"

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        
    }
}
